interfaces/Portalicious/src/app/pages/project-registration-personal-information/components/edit-personal-information/edit-personal-information.component.ts

```diff
@@ -117,6 +118,25 @@ export class EditPersonalInformationComponent
       }),
     ),
   );
+
+  dialogFormGroup = new FormGroup({
+    reason: new FormControl('', {
+      nonNullable: true,
+      // eslint-disable-next-line @typescript-eslint/unbound-method -- https://github.com/typescript-eslint/typescript-eslint/issues/1929#issuecomment-618695608
+      validators: [Validators.required],
+    }),
+  });
+
+  dialogFormFieldErrors = generateFieldErrors<DialogFormGroup>(
+    this.dialogFormGroup,
+    {
+      reason: (control) =>
+        control.errors?.required
+          ? $localize`:@@generic-required-field:This field is required.`
+          : undefined,
```

Comment:
```suggestion
      reason: genericFieldIsRequiredValidationMessage,
```

We already have a helper for this (you need to import it).